README.md

```diff
@@ -75,5 +75,13 @@ Current working directory for logging file paths.
 
 Logger function, called once at the end and once per file if `logFiles` is enabled.
 
+##### `options.includeZero: boolean = false`
+
+Set to true if you want to include results with no rows.
```

Comment:
could even support the same syntax `boolean | string` so you can make a big loud error message instead of setting `onlyZero: true`